internal/logic/logic.go

```diff
@@ -16,31 +16,34 @@ import (
 	"time"
 )
 
-//go:generate mockgen --build_flags=--mod=mod --destination=./../../pkg/mock/mock_logic.go --package=mock github.com/vatsal278/TransactionManagementService/internal/logic TransactionManagementServiceLogicIer
```

Comment:
This should be kept to regenerate the mocks incase of any updates. Please check that mockgen is present.
This is for all the files.

Reply:
done